keyboards/zeix/singa/rubrehaku/matrix_diagram.md

```diff
@@ -0,0 +1,30 @@
+# Matrix Diagram for Rubrehaku
+
+```
```

Comment:
From the PR checklist
> * If the keyboard has multiple electrical/switch layouts:
>   * include a LAYOUT_all which specifies all possible layout positions in the electrical matrix
>   * use alternate layout names for all other possible layouts, preferring community layout names if an equivalent is available (e.g. LAYOUT_tkl_ansi, LAYOUT_ortho_4x4 etc.)

Providing only a `LAYOUT` macro, when this clearly supports multiple layouts would be considered incorrect.